docs/api.md

```diff
@@ -537,3 +537,27 @@ localforage.dropInstance({
 When invoked with no arguments, it drops the "store" of the current instance.
 When invoked with an object specifying both `name` and `storeName` properties, it drops the specified "store".
 When invoked with an object specifying only a `name` property, it drops the specified "database" (and all its stores).
+
+# Multiple Stores
+
+```js
+const dbName = 'databaseName';
+
+// Create table 1 in databaseName
+var tableOne = localforage.createInstance({
+    driver      : localforage.INDEXEDDB,
+    name        : dbName,
+    storeName   : 'tableOne',
+    description : '...'
+});
+
+// Create table 2 in databaseName
+var tableTwo = localforage.createInstance({
+    driver      : localforage.INDEXEDDB,
```

Comment:
Technically this isn't needed, as in WebSQL this will use a table and in localStorage it will namespace the keys.
```suggestion
```